c/src/rtcm3tosbp.c

```diff
@@ -17,20 +17,31 @@
    the current system time, which may not be suitable for pre-recorded
    data.  */
 #include <assert.h>
+#include <getopt.h>
 #include <gnss-converters/rtcm3_sbp.h>
 #include <libsbp/edc.h>
 #include <math.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <swiftnav/edc.h>
 #include <swiftnav/gnss_time.h>
 #include <time.h>
+#include <string.h>
 #include <unistd.h>
+#include <getopt.h>
+
+#include "options.h"
 
 #define SBP_PREAMBLE 0x55
+#define STRCMP_EQ 0
+
 
 static struct rtcm3_sbp_state state;
+static void ParseBiases(char *arg);
```

Comment:
use our standard naming style

```suggestion
static void parse_biases(char *arg);
```